va/validation-authority_test.go

```diff
@@ -332,6 +311,54 @@ func TestSimpleHttp(t *testing.T) {
 	test.AssertEquals(t, invalidChall.Error.Type, core.ConnectionProblem)
 }
 
+func TestSimpleHttpRedirectLookup(t *testing.T) {
```

Comment:
This test covers HTTP redirects, which aren't applicable for DVSNI or DNS. Still, I am investigating to see where we have a CNAME/DNAME redirect test.